tinygrad/renderer/assembly.py

```diff
@@ -263,14 +278,13 @@ def render_local(self, dest, name, size, dtype) -> List[str]:
 
   def render_loop(self, idx, start, label, acc=None) -> List[str]: return [f"mov.u32 {idx}, {start};", f"{label}:"]
 
-  def render_bra(self, b1, pred=None, b2=None) -> List[str]: return [f"@{pred} bra {b1};", f"@!{pred} bra {b2};"] if pred else [f"bra {b1};"]
+  def render_bra(self, b1, pred=None) -> List[str]: return [f"@{pred} bra {b1};"] if pred else [f"bra {b1};"]
 
   def mem_type(self, dtype): return 's8' if dtype.itemsize == 1 else 'b16' if dtype == dtypes.float16 else self.types[dtype]
 
   def render_load(self, loc, dest, dtype, gate=None, alt=None, ss="", offset=0) -> List[str]:
```

Comment:
What does gate do now, this function looks broken. Does it need to have its signature changed?

Reply:
Oh previously it generated 
```
 95     @%pred_pred_0   bra $loop_0;
 96     @!%pred_pred_0  bra $loop_0_exit;
 97 $loop_0_exit:
```
The second jump does nothing so I just removed that - never needed that b2 in the first place